components/breadcrumb/src/Crumb.js

```diff
@@ -62,22 +69,25 @@ export const Crumb = ({ current, href, text, onClick, overrides, ...rest }) => {
 
 Crumb.propTypes = {
 	/**
-	 * The text of the crumb
+	 * Set as the final crumb, linking to the current page
 	 */
 	current: PropTypes.bool,
 
 	/**
-	 * The crumb link href value.
-	 *
-	 * Defaults to '#0'.
+	 * The crumb link href value
 	 */
-	href: PropTypes.string,
+	href: PropTypes.string.isRequired,
 
 	/**
 	 * The text of the crumb
 	 */
 	text: PropTypes.string.isRequired,
 
+	/**
+	 * Component tag, may receive a react-router Link
+	 */
+	tag: PropTypes.node,
```

Comment:
@dominikwilkowski `PropTypes.node` give me a `checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `tag` supplied to `Crumb`, expected a ReactNode.` 🤔

Reply:
Yeah cause your defaultProp is a string.... You should make that a component:
```
const Anchor = (props) => <a {...props} />
```

Reply:
To be clear: you shouldn't use defaultProps here but check if the props is set and then override the anchor. Using a defaultProp here as a string has to fail as we don't want people to just add strings.

Reply:
First on my list was to ask you how to set a default setting a component as you had mentioned

Will set up in code, please hold, will commit shortly

Reply:
Cos I can’t make the value a component (function) and PropTypes.node can be anything... incl string